sections/predictive-search.liquid

```diff
@@ -0,0 +1,58 @@
+{%- if predictive_search.performed -%}
+  <div id="predictive-search-results">
+    {%- if predictive_search.resources.products.size > 0 -%} 
+      <h3 id="predictive-search-products" class="predictive-search__heading">
+        {{ 'templates.search.products' | t }}
+
+        <svg aria-hidden="true" focusable="false" role="presentation" class="spinner" viewBox="0 0 66 66" xmlns="http://www.w3.org/2000/svg">
+          <circle class="path" fill="none" stroke-width="6" cx="33" cy="33" r="30"></circle>
+        </svg>
+      </h3>
+    {%- endif -%}
+    <ul id="predictive-search-results-list" class="predictive-search__results-list list-unstyled" role="listbox" aria-labelledby="predictive-search-products">
```

Comment:
`predictive-search-results-list` is duplicated when searching on the search results page; first in the header and second in the body. This results in duplicated `id`. Will need to adjust to make sure each instance is unique.

Also any other pairing, such as `aria-activedescendant`.